src/decoder/symphonia.rs

```diff
@@ -77,16 +77,23 @@ impl SymphoniaDecoder {
         };
 
         // Select the first supported track
-        let track_id = probed.format.tracks()
+        let track_id = probed
+            .format
+            .tracks()
             .iter()
-            .find(|t| t.codec_params.codec != CODEC_TYPE_NULL).unwrap().id;
-
-        let decode_opts = Default::default();
+            .find(|t| t.codec_params.codec != CODEC_TYPE_NULL)
+            .unwrap()
```

Comment:
This panics if there is not track with a valid codec or if there are no tracks at all. It would be better to return a DecodeError I think. Since a music player (one of the usecases of rodio) might very well run into bad files.

Reply:
I'm not sure what error to return / where to put the error string, as I think it's bad practice to just put it in the file as I did in the most recent commit.